crates/rover-client/src/operations/graph/introspect/schema.rs

```diff
@@ -770,7 +778,7 @@ mod tests {
           model: String
           """
           The class of this starship, such as "Starfighter" or "Deep Space Mobile
-          Battlestation"
```

Comment:
ohhh this quote string should definitely not be stripped ^^ it's a closing one

Reply:
Yes, needed the new version of apollo-encoder you just released :)